vscode-lean4/src/leanclient.ts

```diff
@@ -216,18 +217,18 @@ export class LeanClient implements Disposable {
                     return;
                 },
 
-                didChange: (data, next) => {
-                    next(data);
+                didChange: async (data, next) => {
```

Comment:
this is probably due to the vscode version change, please revert this since Gabriel wants this to be done in a separate PR.